dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPI.java

```diff
@@ -84,6 +84,8 @@ public interface ContentletAPI {
 	 */
 	public Contentlet find(String inode, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException;
 
+	public Contentlet find(String inode, User user, String variantName, boolean respectFrontendRoles) throws DotDataException, DotSecurityException;
```

Comment:
doc me

Reply:
really I found out that I don't need this method 

https://github.com/dotCMS/core/pull/24182/commits/415016d1343d80b9541f87cff8d59f57f9bc2523#diff-fb823c27ba05099d8593a6900c88a586e9e4ceecdf253f3fafc2143d670bc1bcL87

https://github.com/dotCMS/core/pull/24182/commits/5a9c172afc1383539a17e2dea0918506754f7eb3